Linux/sgx/libsgx_tsgxssl/tsgxssl_version.cpp

```diff
@@ -29,8 +29,8 @@
  *
  */
 
```

Comment:
Can we specify the OpenSSL minor version, such as 1.1.1k? Then we can easily know the libs are corresponding to which OpenSSL version.

Reply:
Yes. I will add the minor version of OpenSSL.